src/pdf.js

```diff
@@ -30,11 +30,26 @@ var pdfjsDisplaySVG = require("./display/svg.js");
 let pdfjsDisplayWorkerOptions = require("./display/worker_options.js");
 let pdfjsDisplayAPICompatibility = require("./display/api_compatibility.js");
 
-if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
+if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")) {
+  pdfjsDisplayAPI.setPDFNetworkStreamFactory(async params => {
+    const [{ PDFNetworkStream }, { PDFFetchStream }] = await Promise.all([
+      SystemJS.import("../src/display/network.js"),
+      SystemJS.import("../src/display/fetch_stream.js"),
+    ]);
+
+    if (
+      pdfjsDisplayDisplayUtils.isFetchSupported() &&
+      pdfjsDisplayDisplayUtils.isValidFetchUrl(params.url)
+    ) {
+      return new PDFFetchStream(params);
+    }
+    return new PDFNetworkStream(params);
+  });
+} else if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("GENERIC")) {
```

Comment:
The `typeof PDFJSDev !== "undefined"` condition can be removed here and in the case below since the case above already handles that.

Reply:
Yes, that seems reasonable.